src/Datadog.Trace/Tags.cs

```diff
@@ -197,10 +197,42 @@ public static class Tags
         public const string AzureAppServicesResourceGroup = "aas.resource.group";
 
         /// <summary>
-        /// The site name of the site instance in azure app services where the traced application is running.
+        /// The site name of the site instance in azure where the traced application is running.
```

Comment:
nit: capitalize "Azure" in all the comments in this file.

Reply:
done